enlighten/_basecounter.py

```diff
@@ -163,7 +163,7 @@ class PrintableCounter(BaseCounter):  # pylint: disable=too-many-instance-attrib
     Base class for printable counters
     """
 
-    __slots__ = ('_closed', '_count_updated', 'enabled', '_fill', 'last_update',
+    __slots__ = ('_closed', '_closed_time', '_count_updated', 'enabled', '_fill', 'last_update',
```

Comment:
I'm hesitant to add another variable when _count_updated is already available. It could be renamed to be more general purpose.